Dockerfile

```diff
@@ -1,22 +1,41 @@
 # stage 1 Generate celestia-appd Binary
-FROM golang:1.18-alpine as builder
+FROM docker.io/golang:1.18.10-alpine3.17 as builder
 # hadolint ignore=DL3018
-RUN apk update && apk --no-cache add make gcc musl-dev git
+RUN apk update && apk --no-cache add \
+    gcc \
+    git \
+    make \
+    musl-dev
 COPY . /celestia-app
 WORKDIR /celestia-app
 RUN make build
 
 # stage 2
-FROM alpine:3.17.2
+FROM docker.io/alpine:3.17.2
+
+ARG UID=10001
```

Comment:
nit: might be worth referencing @jrmanes comment about why it is over 10,000 for future reference